edweiss-app/app/(app)/quiz/createQuizPage.tsx

```diff
@@ -190,8 +190,8 @@ const CreateQuizPage: ApplicationRoute = () => {
 					Quiz does not have any exercise yet.
 				</TText>
 
-				<FancyButton icon='add' backgroundColor='blue' onPress={() => addModalRef.current?.present()} ></FancyButton>
-				<AddExerciseModal modalRef={addModalRef} updateExerciseList={addToExerciseList}></AddExerciseModal>
+				<FancyButton icon='add' backgroundColor='blue' onPress={() => addModalRef.current?.present()} />
```

Comment:
Having a button without any text or instruction might be confusing to users. Try to add an indicative text.

Reply:
I will!